src/utils/VoteUtils.js

```diff
@@ -0,0 +1,11 @@
+import {isEmpty} from 'lodash/fp';
+//helper method -> process match data and converts it to a Yes/No algorithm vote result
+export const processMatchData = (matchData) => {
+  if(isEmpty(matchData)){ return 'N/A';};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9404724)
```suggestion
  if(isEmpty(matchData)){ return 'N/A';}
```